ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -77,25 +78,24 @@ export default function OnboardingMetametrics() {
       className="onboarding-metametrics"
       data-testid="onboarding-metametrics"
     >
-      <Typography
-        variant={TypographyVariant.H2}
-        align={TEXT_ALIGN.CENTER}
-        fontWeight={FONT_WEIGHT.BOLD}
+      <Text
+        variant={TextVariant.headingLg}
+        as="h2"
+        align={TextAlign.Center}
+        fontWeight={FontWeight.Bold}
       >
         {t('onboardingMetametricsTitle')}
-      </Typography>
-      <Typography
+      </Text>
+      <Text
         className="onboarding-metametrics__desc"
-        align={TEXT_ALIGN.CENTER}
+        marginTop={2}
```

Comment:
Just curious about where is this style coming from? Previously we did not have margin-top appended